engine/lib/phx/src/audio/audio.rs

```diff
@@ -60,13 +60,46 @@ impl Audio {
         init_volume: f64,
         fade_millis: u64,
     ) -> Box<SoundInstance> {
-        let emitter = self
+        let mut sound_data_clone = sound.sound_data().clone();
+        sound_data_clone.settings.volume = init_volume.into();
+
+        sound_data_clone.settings.fade_in_tween = Some(Tween {
+            start_time: StartTime::Immediate,
+            duration: Duration::from_millis(fade_millis),
+            easing: Easing::Linear,
+        });
+
+        let sound_handle = self
+            .audio_manager
+            .play(sound_data_clone)
+            .expect("Cannot play sound");
+
+        let sound_handle = Rc::new(RefCell::new(sound_handle));
+        let sound_instance = SoundInstance::new(sound_handle, init_volume, None);
```

Comment:
Since we create `sound_handle` here and pass it exclusively to `SoundInstance`, there is no need to wrap it into the `Rx/RefCell`